nginx/frontend.conf

```
@@ -27,6 +27,15 @@ server {
     proxy_set_header X-GU-ID-Geolocation ip:$remote_addr,country:GB,city:Leeds;
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 
+    # some urls coming through as /assets/assets/path/to/file.js for some reason
+    rewrite ^/assets(/assets/.*)$ $1 permanent;
```

Small suggestion, which I find a little easier to parse, as it’s clear that we’re keeping one `/assets/`.

```suggestion
    rewrite ^/assets/assets/(.*)$ /assets/$1 permanent;
```